### PR TITLE
feat: Update text, comment and vote actions in the card group

### LINKED ISSUE #96: [FEATURE]: Update text, comment and vote actions in the card group
## Summary

- Update card group text
- Create, update and delete card group comments
- Add and remove votes

## User story it relates to

#95

### Motivation

We need this to be able to do the same actions that are possible on a normal card

### PR DESCRIPTION
**Pull request by nunocaseiro**
Relates to #96

## Proposed Changes

- Update card group text controller and service methods
- Create, update and delete card group comments controller and service methods
- Add and remove votes controller and service methods

This pull request is intended to close #96

### CODE REVIEW

**Comment by nunocaseiro** [comment]
@all-contributors please add @rpvsilva for code and review

**Comment by nunocaseiro** [comment]
@all-contributors please add @gfdias for code, review, doc, projectManagement

File: backend/src/modules/cards/services/unmerge.card.service.ts [hunk 1]
```diff
@@ -51,13 +51,15 @@ export class UnmergeCardServiceImpl implements UnmergeCardService {
...
-      const newComments = cardGroup.comments.concat(comments);
-      const newVotes = (cardGroup.votes as unknown as string[]).concat(
-        itemVotes as unknown as string[],
+      const items = cardGroup.items.filter(
+        (item) => item._id.toString() !== draggedCardId,
       );
-
-      if (cardGroup.items.length === 1) {
+      if (items.length === 1) {
+        const { text, comments, votes: itemVotes } = items[0];
```
Comment (rpvsilva) [inline backend/src/modules/cards/services/unmerge.card.service.ts:58]:
```suggestion
        const [{ text, comments, votes: itemVotes }] = items;
```

**Review by rpvsilva — CHANGES_REQUESTED** [review]

**Review by rpvsilva — DISMISSED** [review]
Just change that suggestion and LGTM 🔥

**Review by rpvsilva — APPROVED** [review]

**Pull request merged**